back-end/api/quickstart/tests/endpoints/test_followers_list.py

```diff
@@ -15,14 +15,18 @@ def setUp(self):
       Author.objects.create(**get_test_author_fields(i=2))
     ]
 
-    self.receiver_id = "testId"
-    Follow.objects.create(receiver=self.receiver_id, sender=self.senders[0].id, approved=False)
-    Follow.objects.create(receiver=self.receiver_id, sender=self.senders[1].id, approved=False)
+    self.receiver = Author.objects.create(**get_test_author_fields(i=3))
+    
+    self.follows = [
+      Follow.objects.create(receiver=self.receiver, sender=AuthorSerializer(self.senders[0]).data),
+      Follow.objects.create(receiver=self.receiver, sender=AuthorSerializer(self.senders[1]).data)
+    ]
 
   def test_get_all_followers(self):
```

Comment:
Might be good to add a test that gets no followers back. To make sure that an empty list of followers response looks as expected.